<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTextClip_UI</class>
 <widget class="QDialog" name="QTextClip_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel">
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <layout class="QHBoxLayout" name="fontGroup">
     <item>
      <widget class="QFontComboBox" name="font"/>
     </item>
     <item>
      <widget class="QSpinBox" name="fontSize">
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="weightLabel">
       <property name="text">
        <string>Weight:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="weight">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italic">
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="fgColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="color" stdset="0">
        <color>
         <red>85</red>
         <green>170</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="defaultColor" stdset="0">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="bgLabel">
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <layout class="QHBoxLayout" name="lineGroup">
     <item>
      <widget class="QSpinBox" name="lineWidth">
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="lineColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="lineSpace">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QTextEdit" name="text"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="fontLabel">
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="outlineLabel">
     <property name="text">
      <string>Outline</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <layout class="QHBoxLayout" name="bgGroup">
     <item>
      <widget class="KColorButton" name="bgColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="padLabel">
       <property name="text">
        <string>Padding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pad">
       <property name="maximum">
        <number>4096</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="bgSpace">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <layout class="QHBoxLayout" name="durationGroup">
     <item>
      <widget class="QLineEdit" name="duration">
       <property name="inputMask">
        <string>#99:99:99:99</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="durationSpace">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QTextClip_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QTextClip_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
